bee-signing/src/ternary/mod.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 IOTA Stiftung
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and limitations under the License.
+
+//! Ternary signing scheme primitives.
+
+mod constants;
+mod scheme;
+
+pub mod mss;
+pub mod seed;
+pub mod wots;
+
+pub use constants::SIGNATURE_FRAGMENT_LENGTH;
+pub use scheme::{PrivateKey, PrivateKeyGenerator, PublicKey, RecoverableSignature, Signature};
```

Comment:
Could we also export `Seed` here?
```suggestion
pub use scheme::{PrivateKey, PrivateKeyGenerator, PublicKey, RecoverableSignature, Signature};
pub use seed::Seed;
```

Reply:
Whoops! Yes of course.

Reply:
Ah but wait, it is exported though `seed` already.